common-messaging/src/main/resources/application.conf

```diff
@@ -15,7 +15,7 @@ messaging {
     password = ${?NATS_PASSWORD}
   }
   kinesis {
-    appName = "ota-plus"
+    appName = ""
```

Comment:
if there is no app name it should fail, no?

Btw this file should be moved to `reference.conf`. See https://github.com/typesafehub/config#longer-examples